packages/flutter/lib/src/cupertino/context_menu.dart

```diff
@@ -0,0 +1,1222 @@
+///
+///   * [Apple's HIG for Context Menus](https://developer.apple.com/design/human-interface-guidelines/ios/controls/context-menus/)
+class ContextMenu extends StatefulWidget {
+  /// Create a context menu.
+  ContextMenu({
+    Key key,
+    @required this.actions,
+    @required this.child,
+    this.onTap,
+    this.preview,
+  }) : assert(actions != null && actions.isNotEmpty),
+       assert(child != null),
+       super(key: key);
+
+  /// The widget that can be "opened" with the [ContextMenu].
+  ///
+  /// When the [ContextMenu] is long-pressed, the menu will open and this widget
+  /// (or [preview], if provided) will be moved to the overlay and sized to be
+  /// as wide or tall as possible.
+  ///
+  /// When the [ContextMenu] is "closed", this widget acts like a [Container],
+  /// i.e. it does not constrain its child.
+  ///
+  /// This parameter cannot be null.
+  final Widget child;
+
+  /// The actions that are shown in the menu.
+  ///
+  /// This parameter cannot be null, and items in the List must be
+  /// [ContextMenuSheetAction]s.
+  final List<ContextMenuSheetAction> actions;
+
+  /// The callback to call when tapping on the child when the [ContextMenu] is
+  /// open.
+  final VoidCallback onTap;
+
+  /// The widget to show when the [ContextMenu] is open.
+  ///
+  /// If not specified, [child] will be shown.
+  ///
+  /// This can be used to shown an entirely different widget than the child, but
+  /// it can also show a slight variation of the child. As a simple example, the
+  /// child could be given rounded corners in the preview but have sharp corners
+  /// when in the page.
+  ///
+  /// {@tool sample}
+  ///
+  /// Below is an example of using `preview` to create a square image tile very
+  /// similar to the iOS iPhoto app's context menu. When opened, the child
+  /// animates to show its full aspect ratio and has rounded corners.
+  ///
+  /// ```dart
+  /// GridView.count(
+  ///   crossAxisCount: 3,
+  ///   children: <Widget>[
+  ///     ContextMenu(
+  ///       child: FittedBox(
+  ///         fit: BoxFit.cover,
+  ///         child: Image.asset('assets/photo.jpg'),
+  ///       ),
+  ///       // The FittedBox in the preview here allows the image to animate its
+  ///       // aspect ratio when the ContextMenu is animating open and closed.
+  ///       preview: FittedBox(
+  ///         fit: BoxFit.cover,
+  ///         // This ClipRRect rounds the corners of the image when the
+  ///         // ContextMenu is open, even though it's not rounded when it's
+  ///         // closed.
+  ///         child: ClipRRect(
+  ///           borderRadius: BorderRadius.circular(64.0),
+  ///           child: Image.asset('assets/photo.jpg'),
+  ///         ),
+  ///       ),
+  ///       actions: <ContextMenuSheetAction>[
+  ///         ContextMenuSheetAction(
+  ///           child: const Text('Action one'),
+  ///           onPressed: () {},
+  ///         ),
+  ///       ],
+  ///     ),
+  ///   ],
+  /// );
```

Comment:
From the looks of it, this semicolon might be what made the analyzer fail

Reply:
I will try pushing that and see if it works.  Thank you!  I would never have thought of that one.

Reply:
That was it!